readthedocs/api/v2/templates/restapi/footer.html

```diff
@@ -42,7 +42,7 @@
         <dt>{% trans "Versions" %}</dt>
         {% for version in versions %}
         <dd {% if version.verbose_name == current_version %} class="rtd-current-item" {% endif %}>
-          <a href="{{ version.get_subdomain_url }}{{ path|default_if_none:"" }}">{{ version.slug }}</a>
+          <a href="{{ version.get_subdomain_url }}{{ path|default_if_none:"" }}">{{ version.explicit_name }}</a>
```

Comment:
ha, we were displaying the slug here, and the verbose name in the "current version", +1 on using explicit_name for both.